poly-commitment/src/tests/mod.rs

```diff
@@ -0,0 +1,3 @@
+mod batch_15_wires;
```

Comment:
This can be removed. Tests are in `poly-commitment/tests`, not in `poly-commitment/src/tests` anymore.